frontend/src/validation/index.js

```diff
@@ -62,9 +103,42 @@ const visitSchema = Yup.object().shape({
     .integer(),
 })
 
-const validateVisitForm = data => {
+// eslint-disable-next-line no-unused-vars
+const participantSchema = Yup.object().shape({
+  first_name: Yup.string()
+    .required()
+    .max(50),
+  last_name: Yup.string()
+    .required()
+    .max(100),
+  date_of_birth: Yup.string().required(),
+  pp_id: Yup.string()
+    .required()
+    .min(4)
+    .max(200),
+  sep_id: Yup.string()
+    .required()
+    .matches(/[0-9]/),
+  maiden_name: Yup.string()
+    .notRequired()
+    .max(100),
+  is_insured: Yup.boolean().required(),
+  insurer: Yup.string().when("is_insured", {
+    is: false,
+    then: Yup.string().notRequired(),
+    otherwise: Yup.number().required(),
+  }),
+})
+
+const validateForm = (data, inputSchema) => {
   let errors = []
-  return visitSchema
+  let schema
+  if (inputSchema === "visitSchema") {
```

Comment:
i like this.

Can you make constants for the schema names?  `const VISIT_SCHEMA = 'VISIT_SCHEMA'` and then import it from here to the visitRouter/index.